package.json

```diff
@@ -16,9 +16,9 @@
   "main": "./dist/js/okta-sign-in.entry.js",
   "scripts": {
     "clean": "rimraf target && rimraf dist && rimraf build2",
-    "generate-phone-codes": "node buildtools generate-phone-codes",
-    "generate-config": "node buildtools generate-language-config",
-    "update-readme": "node buildtools update-readme",
+    "generate-phone-codes": "node scripts/buildtools generate-phone-codes",
```

Comment:
This script hasn't been run to update this file [since **2019**](https://github.com/okta/okta-signin-widget/commit/5c612e0e5d44ff8ddec9b2b84b661521356c3cd0#diff-b24608fe0e43ebd02010289633e78b1160a4bd0947eb689b63b371d2499ed59d) - can we remove it?

Reply:
This is more one time thing. (leaning towards to keep it now)
There is little gap if we run it now and we just don't have chance to dig deeper.
OKTA-198793